common/app/layout/Breakpoint.scala

```diff
@@ -2,7 +2,7 @@ package layout
 
 import scalaz.syntax.std.option._
 
-sealed trait Breakpoint {
```

Comment:
what happens if you don't break it? I thought sealed just meant you couldn't define Breakpoint instances in any other file.